libs/angular/src/directives/if-feature.directive.spec.ts

```diff
@@ -41,9 +41,9 @@ describe("IfFeatureDirective", () => {
   let content: HTMLElement;
   let mockConfigService: MockProxy<ConfigService>;
 
-  const mockConfigFlagValue = (flag: FeatureFlag, flagValue: FeatureFlagValue) => {
+  const mockConfigFlagValue = (flag: FeatureFlag, flagValue: string | number | boolean) => {
     mockConfigService.getFeatureFlag.mockImplementation((f, defaultValue) =>
-      flag == f ? Promise.resolve(flagValue) : Promise.resolve(defaultValue),
+      flag == f ? Promise.resolve(flagValue as any) : Promise.resolve(defaultValue),
     );
   };
```

Comment:
:art: suggestion: Do we wanna make this type safe?

```suggestion
  const mockConfigFlagValue = <Flag extends FeatureFlag>(flag: Flag, flagValue: FeatureFlagType<Flag>) => {
    mockConfigService.getFeatureFlag.mockImplementation((f, defaultValue) =>
      flag == f ? Promise.resolve(flagValue as any) : Promise.resolve(defaultValue),
    );
  };
```